starter/index.html

```diff
@@ -4,13 +4,13 @@
 <head>
     <meta charset="UTF-8" />
     <link rel="stylesheet" href="./assets/css/style.css">
-    <title>website</title>
```

Comment:
i think this title has to be deleted

Reply:
Thanks. I just changed the title from website to Horiseo.